ananse/commands/binding.py

```diff
@@ -13,7 +13,7 @@
 @logger.catch
 def binding(args):
     predict_peaks(
-        check_path(args.outdir, error_missing=False),
+        check_path(args.outfile, error_missing=False),
```

Comment:
Yes, I thought about that as well. However, if you use a combination of your own peaks as input I think a merged file is currently stored in that directory?